.github/workflows/release.yml

```diff
@@ -0,0 +1,138 @@
+---
+name: "Release"
+on:
+  - "release"
```

Comment:
Do we want to limit the type of release to execute this action on?  https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#release

Reply:
Let's just scope it to `published`:

```
on:
  release:
    types: [published]
```